lib/api/dsl/operators.js

```diff
@@ -207,7 +207,7 @@ module.exports = operators = {
    * Return true only if the point in field is in a specific distance from a geo point
    *
    * @param {String} field the field that we have to check
-   * @param {String} value containing the coordinates from which to calculate the distance and the distance
+   * @param {String|{lat: *, lon: *}|{from: *, to: *}} value containing the coordinates from which to calculate the distance and the distance
```

Comment:
If I read the code correctly, then `value` can never be of type `string`, it was a JSDoc error.
So I guess the correct typing description should be: 
`@param {{lat: *, lon: *}|{from: *, to: *}} value containing the bounding box coords`